test/ContainerResolverTest.php

```diff
@@ -27,9 +27,11 @@ final class ContainerResolverTest extends TestCase
 {
     public function testWillLoadContainerFromInputOption(): void
     {
+        // @codingStandardsIgnoreStart
```

Comment:
What's the CS violation here? If it's "Should be a nowdoc instead of a heredoc" could we instead change to that with `<<<'EOT'`

Reply:
done